src/network/networkd-state-file.c

```diff
@@ -577,6 +599,20 @@ static void serialize_resolvers(
                         fputstrv(f, names, NULL, space);
         }
 
+        if (lease6 && conditional6) {
+                sd_dns_resolver *resolvers;
+                _cleanup_strv_free_ char **names = NULL;
+                int r;
+
+                r = sd_dhcp6_lease_get_dnr(lease6, &resolvers);
+                if (r < 0)
+                        return;
```

Comment:
maybe `return (void) log_warning_errno(r, "Failed to get DNR from DHCPv6 lease, ignoring.")`

or so?

Reply:
(similar above)